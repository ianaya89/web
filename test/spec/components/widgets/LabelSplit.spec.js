import { mount } from '@vue/test-utils'
import LabelSplit from '@/components/widgets/LabelSplit'

describe('LabelSplit', () => {
  it('renders LabelSplit', () => {
    const wrapper = mount(LabelSplit, {
      propsData: {
        values: [
          {
            label: 'Test',
            value: 1
          }
        ]
      }
    })

    expect(wrapper.element).toMatchSnapshot()
  })
})
