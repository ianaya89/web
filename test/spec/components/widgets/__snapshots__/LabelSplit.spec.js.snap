// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LabelSplit renders LabelSplit 1`] = `
<div
  class="w-full flex flex-wrap pt-2"
>
  <div
    class="w-1/2 sm:w-1/4"
  >
    <div>
      <div
        class="text-grey text-p3 uppercase tracking-wide"
      >
        <span
          class="leading-tight"
        >
          Test
        </span>
         
        <!---->
         
        <!---->
      </div>
       
      <div
        class="text-h5 font-medium leading-tight uppercase"
      >
        
      100%
    
      </div>
    </div>
  </div>
</div>
`;
